frontend/src/metabase/components/Popover/Popover.css

```diff
@@ -22,6 +22,12 @@
   overflow: auto;
 }
 
+.tippy-box,
+.tippy-content {
+  max-height: inherit;
+  transition: transform 0s, visibility 0.3s, opacity 0.3s;
```

Comment:
tippy sets `transition-property: transform, visibility, opacity;`, but, interestingly, overriding it with `transition-property: visibility, opacity;` would get rid of the `transform` transition in some browsers but not others. Setting `transition` more explicitly seems to do the trick for all browsers.